CONTRIBUTING.md

```diff
@@ -94,15 +95,17 @@ download and install anything:
 * On each documentation page, there should be an "Improve This Page" link at the very
   top.
 * Click on that link to open the respective source file (usually an `.rst` file in the
-  `doc` folder) on GitHub for editing online (you'll need a GitHub account).
+  `doc/` folder or a `.py` file in the `examples/` folder) on GitHub for editing online
+  (you'll need a GitHub account).
 * Make your desired changes.
 * When you're done, scroll to the bottom of the page.
 * Fill out the two fields under "Commit changes": the first is a short title describing
   your fixes; the second is a more detailed description of the changes. Try to be as
   detailed as possible and describe *why* you changed something.
-* Click on the "Commit changes" button to open a
-  [pull request (see below)](#pull-requests).
-* We'll review your changes and then merge them in if everything is OK.
+* Choose "Create a new branch for this commit and start a pull request" and
+  click on the "Propose changes" button to open a [pull request (see below)](#pull-requests).
```

Comment:
We don't have a title called `pull requests`, right?

Reply:
Fixed in 9296e6878c063ab9729aae9334b0f187365c27b9.